airflow/www/api/experimental/endpoints.py

```diff
@@ -153,24 +153,21 @@ def task_instance_info(dag_id, execution_date, task_id):
     return jsonify(fields)
 
 
-@api_experimental.route('/latest_runs', methods=['GET'])
+@api_experimental.route('/dag_runs/search/<string:query>', methods=['GET'])
```

Comment:
Because the class is called DagRun and python is usually configuration over convention